psalm.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0"?>
+<psalm
+    totallyTyped="false"
+    errorLevel="5"
+    resolveFromConfigFile="true"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xmlns="https://getpsalm.org/schema/config"
+    xsi:schemaLocation="https://getpsalm.org/schema/config vendor/vimeo/psalm/config.xsd"
+>
+    <projectFiles>
+        <directory name="lib/Doctrine/Persistence" />
```

Comment:
not tests?

Reply:
Woops, didn't mean to leave those out

Reply:
Actually, this will be hard to do until 2.0 and until we use phpunit 8, we should probably leave those out.

Reply:
I'm giving a try in case that's not that big of an issue.

Reply:
@bendavies I managed to make it work!  Locally. And now the build Travis fails because phpunit 8 requires php 7.2. So I'm removing these commits, but they are still on a branch on my machine :see_no_evil: